Innovator/Client/scripts/kanban/js/script.js

```diff
@@ -453,16 +459,17 @@ function loadSingleBoard(boardId, reload) {
 
 		// get cardimage
 		var vaultref = item.getProperty("managerpicture");
-		var fileUrl = "nomanager.png";
+		var fileUrl = "../kanban/nomanager.png";
 		if ((vaultref !== undefined) && (vaultRefTable[vaultref] == undefined)) {
-			if (inn.getFileUrl(vaultref.split("?fileId=")[1], 1) == undefined) {
+			try{fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
```

Comment:
Fix the formatting of the try-catch statement.

Reply:
fixed.